uisdk/src/main/java/com/karhoo/uisdk/screen/booking/BookingActivity.kt

```diff
@@ -105,6 +106,11 @@ class BookingActivity : BaseActivity(), AddressBarMVP.Actions, BookingMapMVP.Act
         setNavHeaderImage()
     }
 
+    override fun onPause() {
+        super.onPause()
+        quotesListWidget.cleanup()
```

Comment:
Would it be possible to make the quotesListWidget to be lifecycle aware?

Reply:
Done